bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestBotBuilder.java

```diff
@@ -50,6 +50,7 @@ public class PullRequestBotBuilder {
     private String confOverrideName = ".conf/jcheck";
     private String confOverrideRef = Branch.defaultFor(VCS.GIT).name();
     private String censusLink = null;
+    private boolean enableCsr = true;
```

Comment:
Default should be false.

Reply:
If default is false, we must configurate it as ture in the corresponding projects such as JDK project when deploying. If default is false and we forget to congurate it, the dev flow will be blocked. I am good to change it if you can confirm all the configuration files are changed correctly.

Reply:
Fixed.